src/service/task.go

```diff
@@ -49,3 +59,92 @@ func ReportJudgeTaskResult(
 
 	return nil
 }
+
+func parseVerdictJson(verdictString string) (model.JudgeVerdict, error) {
+	var tests []map[string]interface{}
+	err := json.Unmarshal([]byte(verdictString), &tests)
+	if err != nil {
+		return "", err
+	}
+
+	var (
+		totolTestPoint = len(tests)
+		Priority       = 6
+		AvgTime        = 0.0
+		MaxTime        = 0.0
+		AvgMemory      = 0.0
+		MaxMemory      = 0.0
+	)
+	verdictPriorityMap := map[model.JudgeVerdict]int{
+		model.JudgeVerdictCompileError:        0,
+		model.JudgeVerdictRuntimeError:        1,
+		model.JudgeVerdictTimeLimitExceeded:   2,
+		model.JudgeVerdictMemoryLimitExceeded: 3,
+		model.JudgeVerdictSystemError:         4,
+		model.JudgeVerdictWrongAnswer:         5,
+		model.JudgeVerdictAccepted:            6,
+	}
+	priorityVerdictMap := map[int]model.JudgeVerdict{
+		0: model.JudgeVerdictCompileError,
+		1: model.JudgeVerdictRuntimeError,
+		2: model.JudgeVerdictTimeLimitExceeded,
+		3: model.JudgeVerdictMemoryLimitExceeded,
+		4: model.JudgeVerdictSystemError,
+		5: model.JudgeVerdictWrongAnswer,
+		6: model.JudgeVerdictAccepted,
+	}
+
+	for _, test := range tests {
+		var (
+			tmpVerdict = ""
```

Comment:
This parse func is not good，try to deserialize the JSON by a struct。

Reply:
ok